runtime/Makefile.am

```diff
@@ -89,9 +90,13 @@ nodist_libcsmith_a_include_HEADERS = \
 BUILT_SOURCES = \
 	safe_math.h \
 	safe_math_macros.h \
-	safe_math_macros_notmp.h
+	safe_math_macros_notmp.h \
+	adapter.o \
+	adaptertracking.o
```

Comment:
Not sure if it's good to add object files to BUILT_SOURCES. Perhaps Eric can chime in on this? @eeide